src/index.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
```

Comment:
- [ ] `index.css` file does not exist. Kindly delete line 3 so that your app runs without errors.